File-Formats/PDF/Working-with-DigitalSignature.md

```diff
@@ -3286,42 +2897,38 @@ The following message digest algorithms are now supported:
 * SHA512
 * RIPEMD160
 
-The following code example explains how to add a digital signature with various digest algorithms to the PDF document.
+The following code example explains how to add a digital signature with various digest algorithms to the PDF document using the [DigestAlgorithm](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Security.PdfSignatureSettings.html#Syncfusion_Pdf_Security_PdfSignatureSettings_DigestAlgorithm) enum.
```

Comment:
The following code example explains how to add a digital signature with various digest algorithms to the PDF document by specifying the [DigestAlgorithm]() property as ``SHA256`` through [DigestAlgorithm](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Security.PdfSignatureSettings.html#Syncfusion_Pdf_Security_PdfSignatureSettings_DigestAlgorithm) enum in [PdfSignatureSettings]() class.

Reply:
Changed the content